ReSwiftRouter.xcodeproj/project.pbxproj

```diff
@@ -224,8 +224,7 @@
 				25C0A9CB1C50AF6400139FA7 /* Router.swift */,
 				625E66E21C2000620027C288 /* Frameworks */,
 			);
-			name = ReSwiftRouter;
-			path = SwiftFlowRouter;
```

Comment:
Group reference had SwiftFlowRouter and was showing red in the project. changed to the folder.